scripts/helpers/get_package_path.py

```diff
@@ -1,6 +1,18 @@
 #!/usr/bin/env python3
-from tuda_workspace_scripts.workspace import get_package_path
+from ament_index_python.packages import get_package_share_directory
```

Comment:
You use `ament_index_python` here, but you do not depend on the package in the `package.xml`.

Reply:
True, while it can be expected to be available in any ROS installation, it's cleaner to specify the dependency.
Added this one and a few others.